lib/matplotlib/textpath.py

```diff
@@ -25,19 +31,11 @@ class TextToPath(object):
 
     def __init__(self):
         self.mathtext_parser = MathTextParser('path')
-        self.tex_font_map = None
-
-        from matplotlib.cbook import maxdict
-        self._ps_fontd = maxdict(50)
-
         self._texmanager = None
 
-        self._adobe_standard_encoding = None
-
-    def _get_adobe_standard_encoding(self):
-        enc_name = dviread.find_tex_file('8a.enc')
-        enc = dviread.Encoding(enc_name)
-        return {c: i for i, c in enumerate(enc.encoding)}
+    @cbook.deprecated("3.0")
+    def tex_font_map(self):
```

Comment:
shouldn't this be a property to be backward compatible?

Reply:
yup, fixed